app/index.js

```diff
@@ -16,6 +16,10 @@ var pkg = require('../package.json');
 module.exports = yeoman.generators.Base.extend({
   constructor: function() {
     yeoman.generators.Base.apply(this, arguments);
+    // Skip file conflict detection
+    if (this.conflicter) {
+      this.conflicter.force = true;
+    }
```

Comment:
What is the purpose of this change? I went quite far in the implementation to actually enable conflict detection, as I find it a useful feature. Could you please open a new PR to make & discuss this change?